### PR TITLE
[Backend] Problem in delete contact us api Fixed

### LINKED ISSUE #932: [Backend] Problem in delete contact us api
**Describe the bug**
Delete contact us is not working properly. and contact us routes fixing. and adding validation

**Have you read the [Code of Conduct](https://github.com/HITK-TECH-Community/Community-Website/blob/main/code_of_conduct.md)?**
Write your answer here.
yes

### PR DESCRIPTION
**Pull request by Hemu21**
## Issue that this pull request solves
[Issue Link](https://github.com/HITK-TECH-Community/Community-Website/issues/932) resolve #932 

 Closes: #932 

### Brief description of what is fixed or changed
Delete contact us is not working properly. and contact us routes fixing. and adding validation. fixed. now delete works properly.

## Types of changes

_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] My changes does not break the current system and it passes all the current test cases.

### CODE REVIEW

**Comment by Hemu21** [comment]
@Kajol-Kumari  can you please check and merge it.

File: backend/app/routes/contactUs/delete.js [hunk 1]
```diff
@@ -1,55 +1,26 @@
...
 
-    // Check if contactId is provided
-    if (!contactId || !adminId) {
-        const error = new ErrorHandler(constants.ERRORS.VALIDATION, {
-            statusCode: 400,
-            message: 'Validation Error',
-            errStack: 'Both IDs are required to delete a contact',
-        });
-        return next(error);
+    if (!payload.isSuperAdmin) {
```
Comment (Kajol-Kumari) [inline backend/app/routes/contactUs/delete.js:9]: [ques] why are we doing this check? shouldn't auth middleware take care of it?

Comment (Hemu21) [reply backend/app/routes/contactUs/delete.js:9]: @Kajol-Kumari in every file they is this type of check. So I thought Auth Middleware doesn't handle it.

Comment (Kajol-Kumari) [reply backend/app/routes/contactUs/delete.js:9]: can u provide me the reference of where do u see it, it shouldn't be required ideally, try removing and then test the delete functionality, it should not work without authorization key of an admin

Comment (Hemu21) [reply backend/app/routes/contactUs/delete.js:9]: can you please check this here i saw like this [Here](https://github.com/HITK-TECH-Community/Community-Website/blob/main/backend/app/routes/admin/deleteAdmin.js)

**Review by Kajol-Kumari — APPROVED** [review]

File: backend/app/routes/contactUs/delete.js (hunk 1 above)
Comment (Hemu21) [reply backend/app/routes/contactUs/delete.js:9]: @Kajol-Kumari  middleware is not handdling it. it passing the error or verified data to the next. we have to check. It is needed. you can check the middleware [Here](https://github.com/HITK-TECH-Community/Community-Website/blob/main/backend/helpers/middlewares/auth.js) It throws error only if token is missing.

**Pull request merged**